proxy/vm.go

```diff
@@ -16,14 +16,12 @@ package main
 
 import (
 	"encoding/binary"
-	"errors"
 	"fmt"
-	"math"
 	"net"
 	"os"
 	"sync"
 
-	hyper "github.com/hyperhq/runv/hyperstart/api/json"
+	"github.com/sameo/virtcontainers/hyperstart"
```

Comment:
@jodh-intel The repo name will change, and we will update cc-oci-runtime once it does. We won't close the `sameo/virtcontainers/` repo, we'll transition smoothly to the new one (either at `github.com/01org/` or `github.com/containers`.